oddsgym/envs/tennis.py

```diff
@@ -0,0 +1,59 @@
+import pandas
+from .meta import MetaEnvBuilder
+
+
+class TennisOddsEnv(metaclass=MetaEnvBuilder):
+    sport = 'tennis'
+    versionadded = '0.8.0'
+    odds_column_names = ['win', 'lose']
+
+    def __init__(self, tennis_bets_dataframe):
+        """Initializes a new environment.
+
+        Parameters
+        ----------
+        tennis_bets_dataframe: dataframe of shape (n_matches, n_odds + 3), n_odds == 2
+            A list of matches, with their betting odds, results and the players playing.
+
+            .. warning::
```

Comment:
what happens when it's not this way? would the init fail?

Reply:
Yes, as desiged